bench/single_file_bench.exs

```diff
@@ -1,27 +0,0 @@
-defmodule Dogma.SingleSourceBench do
```

Comment:
It's probably a good idea to test the benchmarks, but I don't think that we can run benchmarks on CI as we don't know if the resources available will be the same on each run.